docs/index.rst

```diff
@@ -4,6 +4,20 @@ coniql
 Control system interface in GraphQL with plugins for EPICS Channel Access and PV Access.
 Supports a web interface to get, put and monitor the value of PVs.
 
+Dependencies
+------------
+Coniql requires Python 3.7+. 
+
+If your system does not have Python 3.7, either use the shared Python install, or build Python3.7 from source. Follow this `guide <https://tecadmin.net/install-python-3-7-on-centos-8/>`_  for RedHat step-by-step, with the following extras::
```

Comment:
Thanks, but I think this section probably goes somewhere in this tree instead:
https://confluence.diamond.ac.uk/display/CNTRLS/Python+3

We try to put details about how to get a particular Python version there as it is site specific, and we have rather a lot of python projects so putting the same info in all of them adds a lot of duplication

Reply:
Thanks, I'm happy to move this section into Confluence. It would probably best fit in the "troubleshooting" page of the page you linked? I don't think it particularly fits into any of the existing child pages.